scout/server/blueprints/institutes/forms.py

```diff
@@ -13,6 +15,25 @@
 from scout.constants import PHENOTYPE_GROUPS
 
 
+SEARCH_LABELS = [
+    "Case or individual name",
+    "HPO term",
+    "Search synopsis",
+    "Gene panel",
+    "Case status",
+    "Phenotype group",
+    "Patient cohort",
+    "Similar case",
+    "Similar phenotype",
+    "Pinned gene",
+    "Causative gene",
+    "Assigned user",
+]
+
+# Generated HTML tag <option>'s 'value' and 'label' will be the same
+CASE_SEARCH_KEY = [(label, label) for label in SEARCH_LABELS]
```

Comment:
If we add a requirement for WTForms >= 2.3. I think my fix could be removed -doesn't matter. 

Otherwise OK!

Reply:
I'm changing that code anyway because of your suggestion but I think it's better to have it as a tuple!

Reply:
No, do as you think is best -keep it as a tuple 

:)